app/assets/stylesheets/navigation.scss

```diff
@@ -29,12 +32,35 @@
 }
 #search_bar {
   width: 115px;
+  border-top: 0px;
+  border-right: 0px;
+  border-left: 0px;
+  box-shadow: none;
+  border-color: #ffffff;
+}
+
+#search_bar:hover {
```

Comment:
Avoid using id selectors<br>Selector `search_bar` should be written in lowercase with hyphens